### DISCUSSION TITLE
Reset Miner on Mh drop

### DISCUSSION THREAD

**Discussion by CCWBee** [Lolliedieb/lolMiner-releases#166] [Q&A]

**Comment by stevepresley** [comment]
I, as well as others, have opened an issue about this.  Here is my particular investigation:

#125 

I am working on a monitor app that will detect this and restart the thread(s) until there is a built-in solution.

**Comment by BeardedAtheist** [comment]
Okay, so here is my code that I used to make it auto restart: `call mine_eth.bat`

Just place this inside of your mine_eth.bat at the end and it auto restarts again.

**Reply by stevepresley** [reply]
> Okay, so here is my code that I used to make it auto restart: `call mine_eth.bat`
> 
> Just place this inside of your mine_eth.bat at the end and it auto restarts again.

your copy/paste didn't work

**Comment by BeardedAtheist** [comment]
Sorry, not sure why unless your BAT file is different.

On Jan 27, 2021, at 4:22 AM, Steve Presley <notifications@github.com> wrote:

﻿

Okay, so here is my code that I used to make it auto restart: call mine_eth.bat

Just place this inside of your mine_eth.bat at the end and it auto restarts again.

your copy/paste didn't work

**Reply by stevepresley** [reply]
> Sorry, not sure why unless your BAT file is different.

Ah, your copy/paste is showing now.

So, the problem is that when the miner goes to 0 MH/s, it never exits, it just loops forever, so just putting "call mine_eth.bat" at the end of the batch file will never get triggered.

I see what you're saying to have it auto-restart, that's the equivalent of:

:loop
lolminer.exe  (plus your arguments)
goto loop

The problem we are having is that the temp shutdown/start doesn't work once it goes to 0 MH/s, so we need something external to monitor either the log file or the API port, detect the 0 MH/s and terminate the process that is still running.

**Comment by BeardedAtheist** [comment]
Maybe this is a work around


echo off
:loop
start yourtarget.exe ...
timeout /t 1200 >null
taskkill /f /im yourtarget.exe >nul
goto loop


On Jan 27, 2021, at 3:06 PM, Steve Presley <notifications@github.com> wrote:

﻿

Sorry, not sure why unless your BAT file is different.

Ah, your copy/paste is showing now.

So, the problem is that when the miner goes to 0 MH/s, it never exits, it just loops forever, so just putting "call mine_eth.bat" at the end of the batch file will never get triggered.

I see what you're saying to have it auto-restart, that's the equivalent of:

:loop
lolminer.exe (plus your arguments)
goto loop

The problem we are having is that the temp shutdown/start doesn't work once it goes to 0 MH/s, so we need something external to monitor either the log file or the API port, detect the 0 MH/s and terminate the process that is still running.
